apps/roboshield/src/components/Footer/FooterDescription.tsx

```diff
@@ -32,18 +33,25 @@ function FooterDescription({ description, logo, sx }: FooterDescriptionProps) {
           }}
         />
       </Link>
-      <RichTypography
-        LinkProps={{
-          color: "text.secondary",
-          sx: { textDecorationColor: "text.secondary" },
+      <RichText
+        variant="footer"
+        typographyProps={{
+          LinkProps: {
+            color: "text.secondary",
+            sx: { textDecorationColor: "text.secondary" },
+          },
         }}
-        sx={{
-          color: "text.secondary",
-          mt: "52px !important",
-        }}
-      >
-        {description}
-      </RichTypography>
+        sx={(theme: Theme) => ({
+          a: {
+            color: theme.palette.text.secondary,
+            textDecorationColor: theme.palette.text.secondary,
+          },
+          mt: "52px",
+          textAlign: { xs: "center", md: "left" },
```

Comment:
```suggestion
```

Even in mobile, paragraph text looks better when left aligned.


![S](https://github.com/CodeForAfrica/ui/assets/1779590/20f8cea0-247b-4816-909e-dfb1e67fc71f)